liblangutil/ErrorReporter.h

```diff
@@ -124,6 +124,9 @@ class ErrorReporter
 	void docstringParsingError(ErrorId _error, std::string const& _description);
 	void docstringParsingError(ErrorId _error, SourceLocation const& _location, std::string const& _description);
 
+	void codeGenerationError(ErrorId _error, std::string const& _description);
+	void fatalCodeGenerationError(ErrorId _error, std::string const& _description);
```

Comment:
These functions are added to the `ErrorReporter` interface but are nowhere used. Removing again? :)

Reply:
> Also, interestingly, I could not reproduce the original report's error anymore. It actually works (maybe some other PR fixed that part implicitly?)

I still could reproduce it. Did you use the `--optimize` flag?

> These functions are added to the ErrorReporter interface but are nowhere used. Removing again? :)

They were used, but not anymore. I decided to use warning, in particular to avoid making a new error category.